CommonMark/Parser/InlineMethods.cs

```diff
@@ -14,11 +14,12 @@ internal static class InlineMethods
         /// Initializes the array of delegates for inline parsing.
         /// </summary>
         /// <returns></returns>
-        internal static Func<Subject, Inline>[] InitializeParsers(CommonMarkSettings settings)
+        internal static Func<Subject, CommonMarkSettings, Inline>[] InitializeParsers(CommonMarkSettings settings)
```

Comment:
OK, I figured a way, but I don't find it very pretty:

https://github.com/AMDL/CommonMark.NET/commit/00420a4eacc349fbd18a325b6ac294d1d856972f